fe/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -808,7 +808,8 @@ private void loadTxnRollbackImpl(TLoadTxnRollbackRequest request) throws UserExc
             checkPasswordAndPrivs(cluster, request.getUser(), request.getPasswd(), request.getDb(),
                     request.getTbl(), request.getUser_ip(), PrivPredicate.LOAD);
         }
-        Database db = Catalog.getInstance().getDb(request.getDb());
+        String dbName = cluster + ":" + request.getDb();
```

Comment:
Use `ClusterNamespace.getFullName()` for safety

Reply:
OK, I fixed it